cr.sh

```diff
@@ -177,6 +177,17 @@ lookup_latest_tag() {
     fi
 }
 
+filter_charts() {
+    while read chart; do
+        local file="$chart/Chart.yaml"
```

Comment:
Since we're filtering anyway we may as well filter out files silently, so we don't get messages like:
> README.md/Chart.yaml is missing, assuming that 'README.md' is not a helm chart

For example:

```suggestion
        [[ ! -d $chart ]] && continue
        local file="$chart/Chart.yaml"
```